editors/neovim-qt/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+PortGroup github 1.0
+
+# use cmake 1.0 for CMAKE_BUILD_TYPE=Release
+PortGroup cmake 1.0
+
+github.setup        equalsraf neovim-qt 0.2.17 v
+revision            1
+categories          editors
+platforms           darwin
+maintainers         {@sstallion gmail.com:sstallion} openmaintainer
+license             ISC
+
+description         Neovim client library and GUI, in Qt5
+
+long_description \
+    Neovim Qt is a lightweight cross-platform Neovim GUI written in C++ with Qt.
+
+homepage            https://github.com/equalsraf/neovim-qt/wiki
+
+checksums           rmd160  5dc73c43c648b8446654773aec515795a9628ed4 \
+                    sha256  d1a86c685c933ab5518ee1505114e9a510816f0db9ff4ac054b40d4b0accfcb1 \
+                    size    1076373
+
+depends_lib         port:neovim \
+                    port:qt5-qtbase \
```

Comment:
You should probably use the "qt5" PortGroup instead, as that will do quite a bit of magic by selecting the appropriate Qt5 version depending on the OS version and such. Check the PortGroup fir more information or check out some Portfiles that use this PG to figure out its use.